src/nxdoc/nuxeo-add-ons/nuxeo-management-rest-api.md

```diff
@@ -0,0 +1,762 @@
+
+```
+org.nuxeo.rest.management.user=transient/technical_user
+```
+
+The user does not need to exist in Nuxeo, and **must** start with `transient/` as we are relying on the transient user feature.
+
+Once you have created the user, configure a JWT secret in nuxeo.conf:
+
+```
+nuxeo.jwt.secret=abracadabra
+```
+
+Then, to use the Management REST API:
+
+- share the JWT secret (`abracadabra` here) between the Nuxeo Server and the client calling the Management REST API
+- generate a JWT token with the user (`transient/technical_user` here) as claim subject
+- call the API using the `Authorization: Bearer JWT_TOKEN` header
+
+### Deploy the Management REST API on a separate port
+
+### Deploy the Management REST API on a separate HTTP port
+
+For security reasons, it is recommended to deploy the Management REST API on a different port from the regular Nuxeo application one. For instance, see below to configure the HTTP port to `9090`.
+
+#### Configure the Management REST API HTTP port
+
+In `nuxeo.conf`, add:
+```
+nuxeo.server.http.managementPort=9090
+```
+
+#### Add a new Tomcat Connector
+
+Add a new `Connector` for the Catalina `Service` in `./conf/server.xml` like the one below:
+
+```xml
+<Connector port="9090" protocol="HTTP/1.1" URIEncoding="UTF-8"
+           address="0.0.0.0"
+           maxThreads="2"
+           acceptCount="10"
+           compression="on"
+           compressionMinSize="512"
+           compressibleMimeType="text/css,application/javascript,text/xml,text/html"
+           connectionTimeout="20000"
+           disableUploadTimeout="false"
+           connectionUploadTimeout="60000" />
+```
+
+Here below the Freemarker version using nuxeo.conf properties:
+
+```xml
+<Connector port="${nuxeo.server.http.managementPort}" protocol="HTTP/1.1" URIEncoding="UTF-8"
+               address="${nuxeo.bind.address}"
+<#if nuxeo.server.signature??>
+               server="${nuxeo.server.signature}"
+</#if>
+               maxThreads="2"
+               acceptCount="10"
+               compression="on"
+               compressionMinSize="512"
+               compressibleMimeType="text/css,application/javascript,text/xml,text/html"
+               connectionTimeout="20000"
+               disableUploadTimeout="false"
+               connectionUploadTimeout="${nuxeo.server.http.connectionUploadTimeout}" />
+```
+
+You can find the default template holding this configuration under `./templates/common-base/conf/server.xml.nxftl` in the Nuxeo Platform.
+
+## Management REST API Endpoints
+
+### Binaries Endpoint
+
+To [Garbage Collect]({{page page='garbage-collecting-orphaned-binaries'}}) the orphaned (unused) binaries as below:
+
+```
+DELETE https://localhost:8080/nuxeo/site/management/binaries/orphaned
+```
+
+**Response**
+
+If successful, returns a JSON representation (see below) of the binary manager status.
+
+**Status Codes**
+
+- 200 *OK* - Success.
+- 409 *Conflict* - Failure.
```

Comment:
Add a meaningful description instead of "Failure".

Reply:
- 409 *Conflict* - Garbage collection is already in progress.